perk-store-webapps/src/main/webapp/vue-app/components/perk-store/ProductDetail.vue

```diff
@@ -16,117 +16,115 @@ Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
 -->
 <template>
   <v-flex class="productDetailContent">
-    <v-hover>
-      <v-card
-        slot-scope="{ hover }"
-        :class="hideElevation && 'productDetailContentCard' || `elevation-${hover ? 9 : 1} productDetailContentCard`"
-        :max-width="cardHeight">
-        <v-carousel
-          :show-arrows="false"
-          :interval="3000"
-          :height="carousselHeight"
-          hide-delimiters
-          cycle
-          class="carousselParent">
-          <template v-if="product.imageFiles">
-            <v-carousel-item
-              v-for="(imageFile,i) in product.imageFiles"
-              :key="i"
-              :src="imageFile.src"
-              max="300"
-              class="carousselImage">
-              <v-toolbar
-                color="transparent"
-                class="toolbarCard"
-                flat>
-                <v-btn
-                  icon
-                  small
-                  :title="$t('exoplatform.perkstore.button.productInfo')"
-                  class="peopleInfoIcon productCardAction d-flex">
-                  <v-icon size="12">fa-info</v-icon>
-                </v-btn>
-                <v-spacer />
-                <v-menu
-                  v-if="userData.canEdit"
-                  v-model="showMenu"
-                  offset-y
-                  attach
-                  :nudge-left="110">
-                  <template v-slot:activator="{ on}">
-                    <v-btn
-                      dark
-                      :title="$t('exoplatform.perkstore.button.editProduct', {0: product.title})"
-                      icon
-                      class="productCardAction mr-0"
-                      v-on="on"
-                      @blur="closeMenu">
-                      <v-icon>mdi-dots-vertical</v-icon>
-                    </v-btn>
-                  </template>
-                  <v-list class="pa-0 ma-0">
-                    <v-list-item class="editLabelProduct" @mousedown="$event.preventDefault()">
-                      <v-list-item-title class="editProductMenu ml-n2" @click="$emit('edit', product)">
-                        <i class="uiIconEdit mr-2"> </i>{{ $t('exoplatform.perkstore.button.menuEditProduct') }}
-                      </v-list-item-title>
-                    </v-list-item>
-                  </v-list>
-                </v-menu>
-              </v-toolbar>
-            </v-carousel-item>
-          </template>
-        </v-carousel>
+    <v-card
+      :class="hideElevation && 'productDetailContentCard' || `elevation-${hover ? 9 : 1} productDetailContentCard`"
+      :max-width="cardHeight"
+      v-on="!canBuyProduct ? { click: displayBuyModal } : {}">
```

Comment:
shouldn't be the opposite here ?
`v-on="canBuyProduct ? { click: displayBuyModal } : {}">`

Reply:
changed to cantBuyproduct